test/samples/meson.build

```diff
@@ -0,0 +1,59 @@
+# Create core video, audio, and subtitle streams to construct files out of.
+video = custom_target('video.mkv',
+    output: 'video.mkv',
+    command: [ffmpeg, '-v', 'error', '-y', '-f', 'lavfi', '-i',
+              'testsrc=duration=2:size=1280x720', '@OUTPUT@'],
+)
+
+audio = custom_target('audio.flac',
+    output: 'audio.flac',
+    command: [ffmpeg, '-v', 'error', '-y', '-f', 'lavfi', '-i',
+              'sine=frequency=1000:duration=2', '@OUTPUT@'],
+)
+
+sub = join_paths(source_root, 'test', 'samples', 'sub.srt')
+
+samples = {
+    'eng_default.mkv':
+        [ffmpeg, '-v', 'error', '-y', '-i', video, '-i', audio, '-f', 'srt', '-i', sub,
```

Comment:
We could also move common part of this command to separate variable to make it more readable. Also `'-c:v', 'copy', '-c:a', 'copy', '-c:s', 'srt'` likely can be replaced with simpler `'-c', 'copy', '-c:s', 'srt'`

Reply:
Simplified